core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -251,7 +252,15 @@ public boolean isMountPoint() {
    * @return the list of file block descriptors
    */
   public List<FileBlockInfo> getFileBlockInfos() {
-    return mFileBlockInfos;
+    return new ArrayList<>(mFileBlockInfos.values());
```

Comment:
would this be called often, and would the list copy be expensive?

Reply:
possibly. I keep both a list and a map now.